knimin/config.txt.example

```diff
@@ -3,9 +3,9 @@ debug = True
 help_email =
 
 [postgres]
-USER =
+USER = postgres
 PASSWORD =
-DATABASE =
+DATABASE = ag_test
```

Comment:
The database made by running `ag_make_test` i called ag_test, so can't really rename.